coffee_maker_navigator_2/lib/di/src/framework/manager/dependency_container_access_handler.dart

```diff
@@ -0,0 +1,56 @@
+/// A contract for managing access to dependency containers, including subscribing to,
+/// unsubscribing from, and resolving container instances.
+///
+/// Implementations of this interface are responsible for adding and removing the subscription of
+/// objects to containers, as well as resolving and providing active instances of
+/// those containers.
+///
+/// This interface ensures that dependency containers are accessible, created, and
+/// disposed of based on their usage within the application, promoting efficient
+/// resource management and modular design.
+abstract interface class DependencyContainerAccessHandler {
```

Comment:
tbh name is a bit verbose, and complicated to get that it is an interface for a manager, sounds like smth related to containers